demos/time_series_demo/python/requirements.txt

```diff
@@ -0,0 +1,3 @@
+matplotlib
+numpy
+pandas
```

Comment:
Pandas doesn't seem to actually be used.

Reply:
done